vuepress/docs/v6.3.2/docs/curate/ecr-bundle-details.md

```diff
@@ -144,47 +144,47 @@ If you are using the `deploymentBaseName` property and want to install more vers
 
 ### Permissions
 
-The `permissions` property specifies a list of coupled clientIds and roles that will be bound in Keycloak. To find them, open the Keycloak console and navigate to _clients_ → _awesomeplugin-server_ → _Service Account Roles_.
+The `permissions` property specifies a list of coupled `clientIds` and roles that will be bound in Keycloak. To find them, open the Keycloak console and navigate to `clients` → `awesomeplugin-server` → `Service Account Roles`.
 ## Widget
 
-Here is an example of a widget descriptor:
+Here is an example of a Widget descriptor:
 
 **Widget descriptor.yaml.**
 
-    code: another_todomvc_widget # The widget identification
+    code: another_todomvc_widget # The Widget identification
 
     titles: # Widget's Titles
       en: TODO MVC Widget # Title in English
       it: TODO MVC Widget # Title in Italian
 
-    group: free # The owner group of the widget
+    group: free # The owner group of the Widget
 
     # Optional. The UI Path, where the widget.ftl file will have the customUi content
     customUiPath: widget.ftl
 
     # Optional. The Custom UI
     customUi: >-
-        <h1>My custom widget UI</h1>
+        <h1>My custom Widget UI</h1>
 
     # Optional. The ConfigUI
     configUi:
-      customElement: todomvc-config # The name of the custom-element used to render the configUI
+      customElement: todomvc-config # The name of the custom element used to render the configUI
       resources:
-        - <bundleid>/static/js/main.js # The resources necessary to the custom element to render the configUI, like the code
+        - <bundleid>/static/js/main.js # The resources necessary for the custom element to render the configUI, like the code
 
 ## Fragment
 
-Here is an example of a fragment descriptor:
+Here is an example of a Fragment descriptor:
 
 **Fragment descriptor.yaml.**
```

Comment:
the periods are odd, unless they are necessary for formatting or something

Reply:
good catch, jinah! i totally glossed over those. maybe i should eat more carrots :P